observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now do more than ever with the Network Observability Operator by using Prometheus metrics and relying less on Loki for storage. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. You can add custom labels for any subnet that you would want to identify in your flows and metrics. This enhancement can also be used to better identify traffic from or to IPs that are external to the cluster. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+With the following updates, you can experience improved performances of the eBPF agent, both in terms of CPU and memory: 
+
+* The eBPF agent now uses TCX webhooks instead of TC.
+* The *NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two cases: when the Network Observability eBPF agent hashmap table is full, which causes the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. 
+* Duplicated flows are now merged into a list, which is tied to a single flow data, so Loki storage demand decreases significantly. Previously, duplicated flows were stored as individual flows, and flagged as duplicate if there were multiple of the same kind of flows. You could choose to show the duplicated flows using the *Duplicate flows* Query Option. Now instead of having multiple, individual flows, there is one flow with a list of matching *Interfaces* and *Interface directions*.
```

Comment:
```suggestion
* Loki storage demand decreases significantly now that duplicated flows are merged into a list, which is tied to a single flow data. Now instead of having multiple, individual flows, there is one flow with a list of matching *Interfaces* and *Interface Directions*. Previously, duplicated flows were stored as individual flows and flagged as duplicates if there was more than one of the same kind of flow. You could choose to show the duplicated flows using the *Duplicate flows* Query Option. 
```
A little org and simplification for your consideration.

Reply:
Borrowing your suggestion and trying to simplify more :-) :

```suggestion
* Loki storage demand decreases significantly now that per-agent duplicated flows are merged. Now instead of having multiple, individual duplicated flows per network interface, there is one de-duplicated flow with a list of related network interfaces.
```
I'm not sure we need to call out the previous "Show duplicates" query option, or if it's clear enough that this was removed because there's no duplicates anymore. I'd say it's a minor detail.

(also: not sure if "de-duplicated" is correct for the doc, or if it's with or without dash)

Reply:
I *think* de-duplicate is OK. IBMSG says "In some words in which identical letters would be next to each other (for example, anti-intellectual, co-opt, de-emphasize)".

Reply:
I landed on this:
Loki storage demand decreases significantly now that duplicated flows are merged into a list, which is tied to a single flow data. Now, instead of having multiple, individual duplicated flows per network interface, there is one de-duplicated flow with a list of related network interfaces.

Reply:
It doesn't look technically correct to say "now that duplicated flows are merged into a list, which is tied to a single flow data"
It's actually the *network interfaces* of the duplicates that are merged, as a list of interfaces, into a single de-duplicated flow. Which is correctly said in the second sentence. If just saying "now that duplicated flows are merged" doesn't look good because you expect a "merged into what?", perhaps we can say differently: "now that  duplicated flows are removed", and the second sentence brings the precisions about the merge of the network interfaces

Reply:
Okay, I updated to say:
Loki storage demand decreases significantly **now that duplicated flows are removed**. Instead of having multiple, individual duplicated flows per network interface, there is one de-duplicated flow with a list of related network interfaces.